packages/matter-node.js/package.json

```diff
@@ -58,6 +58,7 @@
     },
     "files": [
         "dist/**/*",
+        "src/**/*",
```

Comment:
should we publish the ts files realkly in the npm package?

Reply:
I lean that way, yeah, WDYT?  My thinking:

Pluses:  I've found transpiled files in NPMs without the source to be obnoxious.  Stack traces just take you to transpiled code and you have to check out repo to see what's really going on.  Also, it gives users the option to do their own transpiling if they want it transpiled differently.

Negatives:  Makes the package larger.  But next to the size of 2x source maps, type declarations and transpiled files I think this effect is pretty minimal.

Reply:
In fact we do not deliver the tools nor tsconfig and nothing else to "transpile their own" ... so the src is just there for "easier refernce" ... Is it really wortth that? The sourcemaps should still produce helpful messages to match code against the repo.
But if you really want tzo allow custom transpiling or "fix locally" we need to add even more ... and then produce maybe other "issues" bevause it is multiple packages ...

Need to think nd we could discuss in the group

Reply:
ok, lets inlcude them. all fine

Reply:
Added back